HW04/aleksei_zhuchkov/lib/student.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class Student
+  attr_accessor :storage_homework, :storage_notification_for_student, :storage_notification_for_mentor
+  attr_reader :name, :surname
+
+  def initialize(student)
+    @name = student[:name].capitalize
+    @surname = student[:surname].capitalize
+    @storage_homework = []
+    @storage_notification_for_student = []
+    @storage_notification_for_mentor = []
+  end
+
+  def add_homework_to_student_storage(homework)
+    @storage_homework << homework
```

Comment:
Please, correct similar issues everywhere
```suggestion
    storage_homework << homework
```

Reply:
It done.